tests/sanity_test.c

```diff
@@ -639,6 +639,13 @@ int main(int argc, const char* argv[])
     (void) argc;
     (void) argv;
 
+#ifdef __OpenBSD__
```

Comment:
I understand from your comment that this is just meant for testing the following code snippet,
as a mirror of the other instance in `xxhsum`'s `main()`.
However, since we already test `xxhsum` as part of CI and from `make check`,
what do we gain by checking this code snippet a second time in `sanity_test` ?

Consider for example a scenario of a future modification of these lines by some future contributor.
It seems it implies that both instances of the code snippet should be kept in sync manually.

Reply:
Sure, I'll just drop this hunk.